msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/brokerapi/TestCase1561087.java

```diff
@@ -96,4 +101,9 @@ public String getAuthority() {
     public int getConfigFileResourceId() {
         return R.raw.msal_config_default;
     }
+
+    private void containsWithoutBrackets(final String flightToCheck) {
```

Comment:
Can you please add javadoc/comments and use a more descriptive method name? With the current method name it is very hard to know what the method really does